namespaces/live-1.cloud-platform.service.justice.gov.uk/formbuilder-product-page-prod/resources/ecr.tf

```diff
@@ -0,0 +1,27 @@
+terraform {
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = "eu-west-2"
+}
+
+module "formbuilder_product_page_ecr_credentials" {
+  source    = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=2.1"
```

Comment:
In live-1 you can (and should) use the latest version of the module - currently "ref=3.1"